rust/src/utils.rs

```diff
@@ -243,6 +269,27 @@ impl Deserialize for BigNum {
   }
 }
 
+impl serde::Serialize for BigNum {
```

Comment:
Do we want `to_json()`/`to_js_value()`/`from_json()` here (and BigInt, etc) for consistency even though `to_str()`/`from_str()` do the same thing as this would do?